src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:32116/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:32116/project/issues?id=sample&branch=feature1&issues=AWEnPnVj_3Tps8tBBR1N&open=AWEnPnVj_3Tps8tBBR1N)

[ISSUE_KEY]: AWEnPnVj_3Tps8tBBR1N